### ISSUE TITLE
ember-template-lint "overrides" config does not work with eslint-plugin-hbs

### ISSUE THREAD

**Issue by tstewart15** [ember-template-lint/eslint-plugin-hbs#40]
Duplicate of #37 since it was closed and I cannot re-open it.

**Comment by jaydgruber** [comment]
Yeah, this still seems to be an issue. https://github.com/ember-template-lint/ember-template-lint/pull/1495 fixed an issue with parsing the overrides. Now if I log `linter.config.overrides` at: https://github.com/ember-template-lint/eslint-plugin-hbs/blob/master/lib/rules/check-hbs-template-literals.js#L65, the overrides are present. But on https://github.com/ember-template-lint/eslint-plugin-hbs/blob/master/lib/rules/check-hbs-template-literals.js#L78, we are not passing the `filePath` that the [template-linter Node API](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/node-api.md) expects. So it has no way to know to apply the overrides. Looking into this to see to see if I can figure out where to grab the filepath from.

**Comment by jaydgruber** [comment]
Seems like if I just add `filePath: context.getFilename()` to the `linter.verify()` options on #L78, this should work. Will see if I can get a PR up w/ a test.
